JWBestPracticeApps/FeedTableViewController/FeedTableViewController/FeedItem/PlayerItemCell.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  FeedItemCell.swift
+//  FeedTableViewController
+//
+//  Created by Amitai Blickstein on 6/26/22.
+//
+
+import UIKit
+import JWPlayerKit
+
+class PlayerItemCell: UITableViewCell {
+    static let reuseIdentifier = "PlayerItemCell"
+    static let nibName         = "PlayerItemCell"
+            
+    // Provided by viewModel.
+    var item: JWPlayerItem? {
```

Comment:
Awesome. I like this. This is exactly the way I like to configure my table cells using a model such as the JWPlayerItem.